workflow/rocoto_funcs/base.py

```diff
@@ -174,13 +173,15 @@ def wflow_task_begin(self,xmlFile):
     xmlFile.write(text)
 
   def wflow_task_part1(self,xmlFile): #write out part1 which excludes dependencies
+    machine=os.getenv('MACHINE').lower()
     text=f'  <command>{self.dcTaskRes["command"]} &HOMErrfs;</command>\n'
     text=text+f'  <join><cyclestr>{self.dcTaskRes["join"]}</cyclestr></join>\n'
     text=text+f'\n  <jobname><cyclestr>{self.dcTaskRes["jobname"]}</cyclestr></jobname>\n'
     text=text+f'  <account>&ACCOUNT;</account>\n'
     text=text+f'  <queue>&QUEUE_DEFAULT;</queue>\n'
     text=text+f'  <partition>&PARTITION;</partition>\n'
-    text=text+f'  <native>-M "cluster"</native>\n'
+    if machine == "gaea": #cluster only needs to be set on Gaea
```

Comment:
@EricJames-NOAA Sorry I will get back to you on this later.
This is not the way we do it. I will talk with you soon.